nucliadb_node/src/grpc/update.rs

```diff
@@ -43,9 +43,12 @@ pub async fn update_loop(parameters: UpdateParameters, cache: Arc<ShardReaderCac
     loop {
         tokio::time::sleep(parameters.refresh_rate).await;
 
-        let Ok(shards_dir_iterator) = read_dir(&parameters.shards_path) else {
-            error!("Update loop can not read shards directory");
-            break;
+        let shards_dir_iterator = match read_dir(&parameters.shards_path) {
+            Ok(iterator) => iterator,
+            Err(error) => {
+                error!("Update loop can not read shards directory: {error:?}");
+                continue;
```

Comment:
Maybe some sleep would be nice to not log in a loop until writer starts

Reply:
Its in a loop, there is an sleep at the start of it